src/fem/nrf_fem_protocol_api.h

```diff
@@ -0,0 +1,275 @@
+ * and the configuration is preserved between calls).
+ *
+ * If a timer event is provided, i.e. NRF_802154_PA_LNA_EVENT_TYPE_TIMER, the LNA will be configured to (de)activate at the application-configured time gap
+ * before the timer instance reaches the given register_value. The time gap is set via @ref nrf_fem_interface_configure.
+ *
+ * If a radio event is provided, i.e. NRF_802154_PA_LNA_EVENT_TYPE_RADIO, the LNA will be configured to (de)activate when the radio event occurs.
+ *
+ * The function sets up the PPIs and GPIOTE channel to activate LNA for the upcoming radio reception.
+ * LNA pin will be active until deactivated, either by configured deactivation event or using @ref pa_lna_deactivate_now.
+ *
+ * @param[in] p_activate_event   Pointer to the activation event structure.
+ * @param[in] p_deactivate_event Pointer to the deactivation event structure.
+ *
+ * @pre To activate LNA it is required that nrf_fem_interface_configure() has been called first.
+ *
+ * @note If a timer event is provided, the caller of this function is responsible for starting the timer and its shorts.
+ *
+ * @note If a timer event is provided, the caller of this function is responsible for stopping the timer no earlier than the provided compare channel expires.
+ *
+ * @retval   ::NRF_SUCCESS               LNA activate setup is successful.
+ * @retval   ::NRF_ERROR_FORBIDDEN       LNA is currently disabled.
+ * @retval   ::NRF_ERROR_INVALID_STATE   LNA activate setup could not be performed due to invalid or missing configuration parameters
+ *                                       in p_activate_event or/and p_deactivate_event.
+ */
+int32_t nrf_802154_fal_lna_configuration_set(const nrf_802154_fal_event_t * const p_activate_event,
+                                             const nrf_802154_fal_event_t * const p_deactivate_event);
+
+/**
+ * @brief Functions clears up the configuration provided by the @ref nrf_802154_fal_lna_configuration_set function.
+ *
+ * @param[in] p_activate_event   Pointer to the activation event structure.
+ * @param[in] p_deactivate_event Pointer to the deactivation event structure.
+ *
+ * @retval   ::NRF_SUCCESS               LNA activate setup purge is successful.
+ * @retval   ::NRF_ERROR_INVALID_STATE   LNA activate setup purge could not be performed due to invalid or missing configuration parameters
+ *                                       in p_activate_event or/and p_deactivate_event.
+ */
+int32_t nrf_802154_fal_lna_configuration_clear(
+    const nrf_802154_fal_event_t * const p_activate_event,
+    const nrf_802154_fal_event_t * const p_deactivate_event);
+
+/**
+ * @brief Deactivate PA/LNA now.
+ * This function will immediately deactivate the PA/LNA pins.
+ */
+void nrf_802154_fal_deactivate_now(nrf_fal_functionality_t type);
+
+/**
+ * @brief Clean up the configured PA/LNA timer/radio instance, PPI and GPIOTE resources.
```

Comment:
```suggestion
 * @brief Cleans up the configured PA/LNA timer/radio instance, PPI, and GPIOTE resources.
```
Is this `resources of PPI and GPIOTE` or `PPI, and also GPIOTE resources`? The comma responds to this second case.

Reply:
It is both PPI resources and GPIOTE resources. So AFAIU, the comma is not needed, amirite?

Reply:
OK, can we use:
`Cleans up the configured PA/LNA timer/radio instance and resources of PPI and GPIOTE.`